src/thunderbots/shared/constants.h

```diff
@@ -16,10 +16,16 @@ const double BALL_MAX_RADIUS = 0.0215;
 // The maximum speed achievable by our robots, in metres per second.
 // TODO: Determine a more realistic value
 const double ROBOT_MAX_SPEED = 2.0;
+// The maximum angular speed achievable by our robots, in rad/sec
+// TODO: Determine a more realistic value
+const double ROBOT_MAX_ANG_SPEED = 4.0;
 // The maximum acceleration achievable by our robots, in metres per seconds squared.
 // TODO: Determine a more realistic value
 const double ROBOT_MAX_ACCELERATION = 3.0;
-
+// The maximum angular acceleration achievable by our robots, in radians per second
+// squared
+// TODO: Determine a more realistic value
```

Comment:
`TODO` should be associated with a github issue (otherwise this will likely never get addressed).

Reply:
Added Issue #178